scripts/functions.sh

```diff
@@ -246,11 +254,8 @@ function install_plugin() {
   if [[ -f "${DATA_PATH}"/"${EXT}".zip ]];then
      unzip "${DATA_PATH}"/"${EXT}".zip -d /tmp/gs_plugin
      echo -e "\e[32m Enabling ${EXT} for GeoServer \033[0m"
-     if [[ -f /geoserver/start.jar ]]; then
-       cp -r -u -p /tmp/gs_plugin/*.jar /geoserver/webapps/geoserver/WEB-INF/lib/
```

Comment:
I am not sure why `$GEOSERVER_HOME` is not used here but hardcode `/geoserver` is used.

If the hardcode flavor is needed because `$GEOSERVER_HOME` is not defined at that point, I can rollback the change here, let me know.  Or let me know how to get `$GEOSERVER_HOME` to be defined at that point.

Reply:
$GEOSERVER_HOME is defined as an environment variable. So he hardcoded value is actually a mistake and should not be there